plugins/aea-crypto-cosmos/setup.py

```diff
@@ -30,7 +30,7 @@
     license="Apache-2.0",
     description="Python package wrapping the public and private key cryptography and ledger api of Cosmos.",
     packages=find_packages(include=["aea_crypto_cosmos*"]),
-    install_requires=["aea>=0.10.0,<0.11.0", "ecdsa>=0.16", "bech32==1.2.0"],
+    install_requires=["aea>=0.10.0,<0.11.0", "ecdsa>=0.15", "bech32==1.2.0"],
```

Comment:
this was needed for compatibility with PyPI `aea`. Shouldn't hurt, as the latest `ecdsa` available is `0.16.1`, will fix after this release. Opened an issue.